{
	"cve": {
		"CVE_data_meta": {
			"ASSIGNER": "psirt@us.ibm.com",
			"ID": "CVE-2022-43907"
		},
		"data_format": "MITRE",
		"data_type": "CVE",
		"data_version": "4.0",
		"description": {
			"description_data": [
				{
					"lang": "en",
					"value": "IBM Security Guardium 11.4 could allow a remote authenticated attacker to execute arbitrary commands on the system by sending a specially crafted request.  IBM X-Force ID:  240901."
				}
			]
		},
		"problemtype": {
			"problemtype_data": [
				{
					"description": [
						{
							"lang": "en",
							"value": "CWE-78"
						}
					]
				}
			]
		},
		"references": {
			"reference_data": [
				{
					"name": "https://exchange.xforce.ibmcloud.com/vulnerabilities/240901",
					"refsource": "MISC",
					"tags": [
						"VDB Entry",
						"Vendor Advisory"
					],
					"url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/240901"
				},
				{
					"name": "https://www.ibm.com/support/pages/node/7028511",
					"refsource": "MISC",
					"tags": [
						"Patch",
						"Vendor Advisory"
					],
					"url": "https://www.ibm.com/support/pages/node/7028511"
				}
			]
		}
	},
	"impact": {
		"baseMetricV3": {
			"cvssV3": {
				"attackComplexity": "LOW",
				"attackVector": "NETWORK",
				"availabilityImpact": "HIGH",
				"baseScore": 8.8,
				"baseSeverity": "HIGH",
				"confidentialityImpact": "HIGH",
				"integrityImpact": "HIGH",
				"privilegesRequired": "LOW",
				"scope": "UNCHANGED",
				"userInteraction": "NONE",
				"vectorString": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:H/I:H/A:H",
				"version": "3.1"
			},
			"exploitabilityScore": 2.8,
			"impactScore": 5.9
		}
	},
	"configurations": {
		"CVE_data_version": "4.0",
		"nodes": [
			{
				"cpe_match": [
					{
						"cpe23Uri": "cpe:2.3:a:ibm:security_guardium:11.4:*:*:*:*:*:*:*",
						"vulnerable": true
					}
				],
				"operator": "OR"
			}
		]
	},
	"lastModifiedDate": "2023-08-29T05:01Z",
	"publishedDate": "2023-08-27T23:15Z"
}
